scene/2d/tile_map.cpp

```diff
@@ -1793,12 +1793,12 @@ String TileMap::get_configuration_warning() const {
 	return warning;
 }
 
-void TileMap::set_show_collision(bool p_value) {
+void TileMap::set_show_collision(CollisionVisibilityMode p_value) {
```

Comment:
`set_collision_visibility_mode()` and `get_collision_visibility_mode()` would be clearer names since it's not a boolean anymore.